CHANGELOG.md

```diff
@@ -6,9 +6,11 @@
 * Added confidence interval band to auto-correlation plot ([1535](https://github.com/arviz-devs/arviz/pull/1535))
 
 ### Maintenance and fixes
+* Updated `io_pymc3.py` rename dict to follow schema convention([1555](https://github.com/arviz-devs/arviz/pull/1555))
 * Updated `from_cmdstan` and `from_numpyro` converter to follow schema convention ([1541](https://github.com/arviz-devs/arviz/pull/1541) and [1525](https://github.com/arviz-devs/arviz/pull/1525))
```

Comment:
```suggestion
* Updated `from_cmdstan`, `from_numpyro` and `from_pymc3` converters to follow schema convention ([1541](https://github.com/arviz-devs/arviz/pull/1541), [1525](https://github.com/arviz-devs/arviz/pull/1525) and [1555](https://github.com/arviz-devs/arviz/pull/1555))
```

Sorry if I was not clear enough before, given that this is the same change done to multiple functions, I think that keeping track of it in a single bullet point of the checklist will be more clear. 

Note that this is what github calls a suggestion, so there is no need to copy paste that into the file locally, you should see a button on this same comments to apply the suggestion

Reply:
I was confused @OriolAbril .I will keep this in mind from now on for new PR's